tests/rptest/tests/controller_availability_test.py

```diff
@@ -85,9 +85,15 @@ def test_controller_availability_with_nodes_down(self, cluster_size, stop):
         to_kill = self._tolerated_failures(
             cluster_size) if stop == "minority" else 1
 
-        for i in range(0, to_kill):
-            self.logger.info(f"stopping node {i}")
-            self.redpanda.stop_node(self.redpanda.nodes[i], forced=True)
+        # stop first two nodes with the highest priorities (this way we guarantee that)
+        nodes = sorted(self.redpanda.nodes.copy(),
```

Comment:
am I understanding it right that the issue here is  the delay caused by target priority to decay if we kill randomly? and sometimes it takes more than timeout.

Reply:
Exactly, especially with debug builds